README.md

```diff
@@ -42,28 +42,43 @@ a default build, you will additionally need libxml2 and libsqlite3.
 
 On Ubuntu, you can install these using:
 
-    sudo apt install -y pkg-config build-essential autoconf bison re2c \
-                        libxml2-dev libsqlite3-dev
+```shell
+sudo apt install -y pkg-config build-essential autoconf bison re2c libxml2-dev libsqlite3-dev
+```
+
+On MacOS, you can install these using:
+
+```shell
+brew install autoconf bison re2c iconv
```

Comment:
Is this enough for a default build with no arguments? E.g. `libxml2` might be needed for `ext-xml`.

Reply:
Just tried to install all of this for the first time and it failed. Tried to install only these and `configure` and `make`  succeeded

Reply:
It indeed needs libxml2 because the xml extensions (except soap and xsl) are all default-enabled. You likely already had libxml2 on your system.

Reply:
Same for libsqlite3.

Reply:
As macos user, I confirm these are necessary.

Reply:
Added these two deps